cli/src/main/java/com/devonfw/tools/ide/tool/ide/IdeaBasedIdeToolCommandlet.java

```diff
@@ -70,6 +72,13 @@ public void installPlugin(ToolPluginDescriptor plugin, Step step) {
     }
   }
 
+  @Override
+  public void runTool(String... args) {
+    List<String> extendedArgs = new ArrayList<>(Arrays.asList(args));
+    extendedArgs.add(this.context.getWorkspacePath().toString());
+    super.runTool(extendedArgs.toArray(new String[0]));
+  }
```

Comment:
I moved the workspace configuration into this location. So when the tool runs, then the workspace is added as an argument.

Reply:
IMHO there is `runIde` for IDE commandlets that could override for this use-case what would make it a little bit more obvious.

Reply:
I can only find `run`, which is final. So I guess we need to keep `runTool` here.

Reply:
It was present once:
https://github.com/devonfw/IDEasy/blob/12cdd6406aea09a8f451592f3abcba57a5ee9b67/cli/src/main/java/com/devonfw/tools/ide/tool/ide/IdeToolCommandlet.java#L56

However, someone removed it meanwhile so we need to stay with this approach.